src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -50,7 +51,17 @@ public function ask(InputInterface $input, OutputInterface $output, Question $qu
         }
 
         if (!$input->isInteractive()) {
-            return $question->getDefault();
+            if (!$question instanceof RepeatedQuestion) {
+                return $question->getDefault();
+            }
+
+            $responses = array();
+            while ($question->shouldContinue()) {
+                $question->addAnswer($question->getDefault());
```

Comment:
We can maybe move this logic to `SymfonyStyle` or `QuestionHelper` and remove `RepeatedQuestion`. Would you prefer?